rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -52,27 +52,17 @@ class MasterGraph
     ~MasterGraph();
     Status reset();
     size_t remaining_count();
-    MasterGraph::Status copy_out_tensor(void *out_ptr, RocalTensorFormat format, float multiplier0, float multiplier1, float multiplier2,
-                    float offset0, float offset1, float offset2, bool reverse_channels, RocalTensorDataType output_data_type);
-    Status copy_output(unsigned char* out_ptr, size_t out_size_in_bytes);
-    Status copy_out_tensor_planar(void *out_ptr, RocalTensorFormat format, float multiplier0, float multiplier1, float multiplier2,
-                    float offset0, float offset1, float offset2, bool reverse_channels, RocalTensorDataType output_data_type);
-    size_t output_width();
-    size_t output_height();
-    size_t output_byte_size();
-    size_t output_depth();
+    rocalTensorList *get_output_tensors();
+    std::vector<size_t> tensor_output_byte_size();
```

Comment:
Can we change this to return the vector as a reference?
std::vector<size_t>& tensor_output_byte_size();

Reply:
Removed this API as it is not used anywhere